src/java.base/share/classes/java/net/ServerSocket.java

```diff
@@ -261,7 +261,7 @@ public ServerSocket(int port, int backlog) throws IOException {
      *             the specified range of valid port values, which is between
      *             0 and 65535, inclusive.
      *
-     * @see SocketOptions
+     * @see StandardSocketOptions
```

Comment:
What would you think about dropping this link, and the link to SocketImpl from all the constructors, they aren't relevant for anyone reading this part of the docs.

Reply:
These constructors talk about the "{@code createSocketImpl} method", so I'm guessing the `@see java.net.SocketImpl` was meant to provide reference to the `SocketImpl`. But I think we should just change the "{@code createSocketImpl} method" to "{@link SocketImplFactory#createSocketImpl()} method" and remove these `@see java.net.SocketImpl`.

I guess we could do the same and remove the `@see java.net.SocketImpl` from the constructors of `java.net.Socket` too?

 I see a couple of similar `@see        java.net.SocketImpl` in the constructors of `java.net.Socket` as well. Should we remove those too?